mne/minimum_norm/resolution_matrix.py

```diff
@@ -63,6 +94,63 @@ def make_inverse_resolution_matrix(
     invmat = _get_matrix_from_inverse_operator(inv, fwd, method=method, lambda2=lambda2)
     resmat = invmat.dot(leadfield)
     logger.info("Dimensions of resolution matrix: %d by %d." % resmat.shape)
+
+    # add source noise power to columns of resolution matrix
+    if noise_cov is not None:
+        inv = prepare_inverse_operator(inv, 1, lambda2, method, copy="non-src")
+        info = _prepare_info(inv)
+        # compute source noise power
+        stc = apply_inverse_cov(
+            noise_cov,
+            info,
+            inv,
+            nave=1,
+            lambda2=lambda2,
+            method=method,
+            pick_ori=None,
+            prepared=True,
+            label=None,
+            method_params=None,
+            use_cps=True,
+            verbose=None,
+        )
+
+        # output will be square-root of power, so take intensities before
+        # adding noise
+        shape = resmat.shape
+        if not shape[0] == shape[1]:
+            # pool loose/free orientations
+            resmat = _rectify_resolution_matrix(resmat)
+        else:
+            resmat = np.abs(resmat)
+
+        # scaling factor
+
+        idx_ch_types = {}
+        idx_ch_types["eeg"] = pick_types(info, eeg=True, meg=False)
+        idx_ch_types["mag"] = pick_types(info, eeg=False, meg="mag")
+        idx_ch_types["gra"] = pick_types(info, eeg=False, meg="grad")
```

Comment:
Would there ever be a scenario that there would be other sensor types as well? Perhaps not.

Reply:
I don't know - are there other sensor types? How are OPMs dealt with?
Is there a method to get all sensor types from info and loop through them?